### PR TITLE
Parallel category V4

### PR DESCRIPTION
**Pull request by Mahdis-z**
Doc upgrade checklist:

- [x] file has been moved from `unconverted/x/y.md` to `x/y.md`
- [x] old boilerplate at top and bottom of file has been removed
- [x] Every example is independently runnable and is optimized for short line count
- [x] no more `plot()` or `iplot()`
- [x] `graph_objs` has been renamed to `graph_objects`
- [X] `fig = <something>` call is high up in each example
- [x] minimal creation of intermediate `trace` objects
- [x] liberal use of `add_trace` and `update_layout`
- [ ] `fig.show()` at the end of each example
- [x] `px` example at the top if appropriate
- [ ] minimize usage of hex codes for colors in favour of those in https://github.com/plotly/plotly.py-docs/issues/14

### CODE REVIEW

**Comment by Mahdis-z** [comment]
closes https://github.com/plotly/plotly.py-docs/issues/104

File: python/parallel-categories-diagram.md [hunk 1]
```diff
@@ -26,66 +36,64 @@ jupyter:
...
+import plotly.express as px
 
-#### Parallel Categories Diagram
-The parallel categories diagram is a visualization of multi-dimensional categorical data sets.  Each variable in the data set is represented by a column of rectangles, where each rectangle corresponds to a discrete value taken on by that variable.  The relative heights of the rectangles reflect the relative frequency of occurrence of the corresponding value.
-
-Combinations of category rectangles across dimensions are connected by ribbons, where the height of the ribbon corresponds to the relative frequency of occurrence of the combination of categories in the data set.
+tips = px.data.tips()
+fig = px.parallel_categories(tips, dimensions=['sex', 'smoker', 'day'], 
+                color="size", color_continuous_scale=px.colors.sequential.Inferno,
+                labels={'sex':'SEX', 'smoker':'SMOKER', 'day':'DAY'})
```
Comment (nicolaskruchten) [inline python/parallel-categories-diagram.md]: Let's do something more interesting than just capitalize them ... how about "Payer Sex", "Smokers at the table" and "Day of week"?

File: python/parallel-categories-diagram.md [hunk 2]
```diff
@@ -26,66 +36,64 @@ jupyter:
...
-                    'Brown', 'Blue', 'Blue', 'Blue']},
+         'values': ['Brown', 'Brown', 'Brown', 'Brown', 'Brown', 'Blue', 'Blue', 'Blue']},
         {'label': 'Sex',
-         'values': ['Female', 'Female', 'Female', 'Male',
-                    'Female', 'Male', 'Male', 'Male']}]
+         'values': ['Female', 'Female', 'Female', 'Male', 'Female', 'Male', 'Male', 'Male']}]
 )
 
-iplot([parcats])
+fig = go.Figure(parcats)
```
Comment (nicolaskruchten) [inline python/parallel-categories-diagram.md]: you don't need the intermediate variable here, please just inline the trace in the figure.

File: python/parallel-categories-diagram.md [hunk 3]
```diff
@@ -97,7 +105,8 @@ parcats = go.Parcats(
     counts=[6, 10, 40, 23, 7]
 )
 
-iplot([parcats])
+fig = go.Figure(parcats)
```
Comment (nicolaskruchten) [inline python/parallel-categories-diagram.md]: again, please inline.

**Comment by nicolaskruchten** [comment]
Looks great!
💃

**Pull request merged**